modules/light-clients/11-beefy/types/update.go

```diff
@@ -191,10 +191,7 @@ func (cs *ClientState) verifyHeader(
 	return nil
 }
 
-func (cs *ClientState) getMMRProf(beefyHeader *Header) (*mmr.Proof, error) {
-	// for debug purposes
-	var mmrLeavesRaw = make([]BeefyMmrLeaf, len(beefyHeader.ParachainHeaders))
-
+func (cs *ClientState) parachainHeadersToMmrProof(beefyHeader *Header) (*mmr.Proof, error) {
```

Comment:
shouldn't it be parachainHeadersToMMRProof ?

Reply:
Mmr is the naming convention.

Reply:
he means that in Go, the convention is to capitalize acronyms, such as MMR

Reply:
I'll update the method name and resolve this